src/main/java/com/aws/greengrass/util/GreengrassServiceClientFactory.java

```diff
@@ -158,8 +165,9 @@ public GreengrassV2DataClient getGreengrassV2DataClient() {
     /**
      * Initializes and returns GreengrassV2DataClient.
      * @throws DeviceConfigurationException when fails to validate configs.
+     * @throws TLSAuthException when fails to configure the client
      */
-    public GreengrassV2DataClient fetchGreengrassV2DataClient() throws DeviceConfigurationException {
+    public GreengrassV2DataClient fetchGreengrassV2DataClient() throws DeviceConfigurationException, TLSAuthException {
```

Comment:
It looks like both `getGreengrassV2DataClient` and `fetchGreengrassV2DataClient` are doing the same thing. Why are there 2 different methods for the same thing?